generate_R_table.py

```diff
@@ -68,8 +69,13 @@ def main():
     # answers to all the questions
     # Key = item number;
     # value = (correct_button_code, LeftorRight)
-    answer_key = dict_from_table(read_table(file_names['Question key filename']),
-                                                    paired=False)
+    
+    # THE TWO LINES BELOW ARE COMMENTED BECAUSE THEY DON'T WORK CORRECTLY.
+    # THE LINE BELOW THEM IS A QUICK FIX (ALSO SEE LINE 396). 
+    #answer_key = dict_from_table(read_table(file_names['Question key filename']),
+    #                                               paired=False)
```

Comment:
While you're in PDB, you can either convert the generator to a list or call `next()` on it to get just one item.
As for the bug report, it had to do with empty lines in DEL files, I don't think that's related to Staub's stuff. Here's the commit that aims to fix it: 2f779743c22a7